testing/tests/DevExpress.ui.widgets.editors/numberBoxParts/common.tests.js

```diff
@@ -1663,6 +1663,32 @@ QUnit.module('regressions', {
 
         assert.ok(!SpinButton.getInstance($spinButton).option('disabled'), 'spin button disabled state is correct');
     });
+
+    QUnit.test('tabindex attribute should be exists after initialization (T1090255)', function(assert) {
```

Comment:
this test is not related to the ticket issue => it's not necessary to add the ID

Reply:
https://github.com/DevExpress/DevExtreme/pull/21796/commits/d8551cbe8adf8266953d361e37319d330d9c0897